src/analytics/selectors.ts

```diff
@@ -93,6 +94,7 @@ export const getCurrentUserTraits = createSelector(
     pincodeTypeSelector,
     superchargeInfoSelector,
     (_state: RootState, networkIds: NetworkId[]) => networkIds,
+    pointsBalanceSelector,
```

Comment:
nit: so it reads more organized? 😄 
```suggestion
    pointsBalanceSelector,
    (_state: RootState, networkIds: NetworkId[]) => networkIds,
```